apps/web/app/[lang]/sites/[id]/page.tsx

```diff
@@ -1,24 +1,64 @@
-// import { Metadata } from 'next';
+import { Metadata } from 'next';
 import { notFound } from 'next/navigation';
-// import { headers } from 'next/headers';
+import { Link } from '@intl/navigation';
+import { ArrowBackIcon, CheckIcon } from 'ui/core/icons';
+import { headers } from 'next/headers';
 import { Layout } from '@components/core/layout/Layout';
-import { defaultLocale } from 'intl';
+import { getTranslations, defaultLocale } from 'intl';
 import { SiteProps } from './Site.types';
 import { getSiteById } from '@api/sites/requests';
+import { H3, Tag, H4, L1 } from 'ui/core';
+import { ImageWithFallback } from '@components/utils/ImageWithFallback';
+import { studyStatus } from './constants';
+import { setMetadata } from './helpers';
 
 const Page = async ({ params: { lang = defaultLocale, id } }: SiteProps) => {
   const site = await getSiteById(id);
   if (!site) notFound();
-
+  const t = await getTranslations({ lang, namespace: 'sites' });
+  const conditionsList = [];
+  site.studies
+    .filter((site) => site.status === studyStatus.RECRUITING)
+    .map((st) =>
+      st.translations
+        .find((trans) => trans[lang])
+        [lang].conditions_ct.map((cond) => {
+          if (conditionsList.indexOf(cond) === -1) {
+            conditionsList.push(cond);
+          }
+        }),
+    );
   return (
     <Layout locale={lang}>
-      <div className="grid grid-cols-1 lg:grid-cols-3 xl:grid-cols-4">
-        <div
-          className="w-full col-span-1 lg:col-span-2 xl:col-span-3 pr-0 lg:pr-12"
-          style={{ backgroundColor: 'yellow' }}
-        >
-          <p>Titulo y descripcion</p>
+      <div className="grid grid-cols-1 lg:grid-cols-3 xl:grid-cols-4 gap-14">
+        <div className="w-full col-span-1 lg:col-span-2 xl:col-span-3 pr-0">
+          <div className="justify-between items-center inline-flex h-12">
+            <Link href="/" locale={lang}>
+              <ArrowBackIcon />
+            </Link>
+          </div>
+
+          <div className="h-full flex-col flex gap-4">
+            <H3
+              label={site.name}
+              className="my-0 sm:my-0 lg:my-0 xl:my-0 !pb-0"
+            />
+            {site.studies.some(
+              (study) => study.status === studyStatus.RECRUITING,
+            ) && (
+              <div className="justify-start items-start gap-2 inline-flex">
+                <Tag
+                  text={t('recruiting')}
+                  className="p-2 px-4 bg-stone-200 uppercase rounded-full justify-end items-center inline-flex text-dark text-xs font-medium font-['DMSans] leading-none whitespace-nowrap"
+                />
+              </div>
+            )}
+            <p className='text-dark font-normal font-["DMSans"] leading-relaxed '>
```

Comment:
Make sure `description` is mandatory in the backend model. If not,
```suggestion
            {site.description && <p className='text-dark font-normal font-["DMSans"] leading-relaxed'>}
```

Also, please use a Typography component